src/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -1929,7 +1930,11 @@ private function getTypes(string $field, mixed $value, ClassMetadata $class): ar
         return $types;
     }
 
-    /** @psalm-return ArrayParameterType::* */
+    /** @psalm-return ArrayParameterType::*
```

Comment:
This is a weird mix of single line and multiline syntax.

Reply:
fixed